include/crow/json.h

```diff
@@ -1851,9 +1851,9 @@ namespace crow
                             } f_state;
                             char outbuf[128];
 #ifdef _MSC_VER
-                            sprintf_s(outbuf, sizeof(outbuf), "%f", v.num.d);
+                            sprintf_s(outbuf, sizeof(outbuf), "%.*g", DBL_DECIMAL_DIG, v.num.d);
 #else
-                            snprintf(outbuf, sizeof(outbuf), "%f", v.num.d);
+                            snprintf(outbuf, sizeof(outbuf), "%.*g", DBL_DECIMAL_DIG, v.num.d);
```

Comment:
As we have set C++ 11, build fails with it, you are right. means we have to use DECIMAL_DIG here.